x/cdp/simulation/decoder.go

```diff
@@ -1,12 +1,60 @@
 package simulation
 
 import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"time"
+
 	"github.com/cosmos/cosmos-sdk/codec"
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	cmn "github.com/tendermint/tendermint/libs/common"
+
+	"github.com/kava-labs/kava/x/cdp/types"
 )
 
 // DecodeStore unmarshals the KVPair's Value to the corresponding cdp type
 func DecodeStore(cdc *codec.Codec, kvA, kvB cmn.KVPair) string {
-	// TODO implement this
-	return ""
+	switch {
+	case bytes.Equal(kvA.Key[:1], types.CdpIDKeyPrefix):
+		var cdpIDsA, cdpIDsB []uint64
+		cdc.MustUnmarshalBinaryLengthPrefixed(kvA.Value, &cdpIDsA)
+		cdc.MustUnmarshalBinaryLengthPrefixed(kvB.Value, &cdpIDsB)
+		return fmt.Sprintf("%v\n%v", cdpIDsA, cdpIDsB)
+
+	case bytes.Equal(kvA.Key[:1], types.CdpIDKey),
```

Comment:
Is there a reason you didn't include `types.CdpKeyPrefix`?

Reply:
maybe I missed it? I'll double check